### PR TITLE
Finish onboarding questions

### PR DESCRIPTION
**Pull request by ruibrii**

### CODE REVIEW

File: dev/readthedocs.md [hunk 1]
```diff
@@ -5,27 +5,55 @@ answer the following questions.
...
+    `semesterly/sensitive.py`
+
 5. What is our stack?
 
+    | Component          | Technology  | Style/Methodology |
+    |--------------------|-------------|-------------------|
+    | Database           | PostgreSQL  | Django ORM        |
+    | Backend Framework  | Django      | pycodestyle/Black |
+    | Frontend Framework | React/Redux | ESLint/Prettier   |
+    | CSS Framework      | SCSS        | BEM/Airbnb        |
```
Comment (JJamesWWang) [inline dev/readthedocs.md:29]: Lol copied this straight for the rst file didn't you

**Review by JJamesWWang — APPROVED** [review]
Looks good!

**Comment by JJamesWWang** [comment]
Actually, it seems like you made a PR to your own repository than the upstream repository.

File: dev/readthedocs.md (hunk 1 above)
Comment (ruibrii) [reply dev/readthedocs.md:29]: yessir

**Pull request closed without merging**